test/cfg/gnu.c

```diff
@@ -164,15 +164,15 @@ void valid_code(int argInt1, va_list valist_arg, int * parg)
     // cppcheck-suppress unreadVariable
     i64_2 = __builtin_bswap64(i64_1++);
     
-    // TODO cppcheck-suppress zerodiv
+    // cppcheck-suppress zerodiv
     // cppcheck-suppress unreadVariable
     i16_1 /= bswap_16(0x1234) - 0x3412;
-    // TODO cppcheck-suppress zerodiv
+    // cppcheck-suppress zerodiv
     // cppcheck-suppress unreadVariable
     i32_1 /= bswap_32(0x12345678) - 0x78563412;
-    // TODO cppcheck-suppress zerodiv
+    // cppcheck-suppress zerodiv
     // cppcheck-suppress unreadVariable
-    i64_1 /= bswap_64(0x123456789abcdef0) - 0xf0debc9a78563412;
+    i64_1 /= bswap_64(0x023456789abcde0f) - 0x0fdebc9a78563402;
```

Comment:
I changed the values in the last test, because the original still fails to detect the zerodiv, but that seems unrelated to the implementation of `bswap_64`.